engine/src/from_cudf/cpp_src/bitmask/null_mask.cu

```diff
@@ -0,0 +1,514 @@
+/*
```

Comment:
oh god do we really need this?

Reply:
This is just utility code i copied from cudf. We need some of the utility code and the utility code has dependencies. It was easier for me to copy the whole folder.